### PR TITLE
Fix Packet guide to comply with latest version of the packet module

### PR DESCRIPTION
**Pull request by t0mk**
##### SUMMARY
This PR fixes the Packet Guide doc to follow the latest merged changes in the packet_device module.

##### ISSUE TYPE
 - Docs Pull Request

##### COMPONENT NAME
docs/docsite/rst/guide_packet.rst

##### ANSIBLE VERSION

```
ansible 2.5.0 (fix-packet-guide-to-comply-with-latest-device-module acdda6f020) last updated 2017/10/06 15:03:35 (GMT +300)
  config file = None
  configured module search path = [u'/home/tomk/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /home/tomk/ansible/lib/ansible
  executable location = /home/tomk/ansible/bin/ansible
  python version = 2.7.12 (default, Nov 19 2016, 06:48:10) [GCC 5.4.0 20160609]
```

### CODE REVIEW

File: docs/docsite/rst/guide_packet.rst [hunk 1]
```diff
@@ -185,7 +185,7 @@ As with most Ansible modules, the default states of the Packet modules are idemp
 
 The second module call provisions 3 Packet Type 0 (specified using the 'plan' parameter) servers in the project identified via the 'project_id' parameter. The servers are all provisioned with CoresOS beta (the 'operating_system' parameter) and are customized with cloud-config user data passed to the 'user_data' parameter.
 
-The ``packet_device`` module has a boolean 'wait' parameter that defaults to 'false'. If set to 'true', Ansible will wait until the GET API call for a device will contain an Internet-routeable IP address. The 'wait' parameter allows us to use the IP address of the device as soon as it's available.
+The ``packet_device`` module has a `wait_for_public_IPv` parameter. If set to `4` or `6`, Ansible will wait until the GET API call for a device will contain an Internet-routeable IP addressi of the respective version. If we want to refer to an IP address in consequent module calls, it's wise to use the `wait_for_public_IPv` parameter.
```
Comment (gundalow) [inline docs/docsite/rst/guide_packet.rst]: In RST it's double for inline code
```
``wait_for_public_IPv``
```

Comment (gundalow) [reply docs/docsite/rst/guide_packet.rst]: typo addressi

File: docs/docsite/rst/guide_packet.rst [hunk 2]
```diff
@@ -185,7 +185,7 @@ As with most Ansible modules, the default states of the Packet modules are idemp
 
 The second module call provisions 3 Packet Type 0 (specified using the 'plan' parameter) servers in the project identified via the 'project_id' parameter. The servers are all provisioned with CoresOS beta (the 'operating_system' parameter) and are customized with cloud-config user data passed to the 'user_data' parameter.
 
-The ``packet_device`` module has a boolean 'wait' parameter that defaults to 'false'. If set to 'true', Ansible will wait until the GET API call for a device will contain an Internet-routeable IP address. The 'wait' parameter allows us to use the IP address of the device as soon as it's available.
+The packet_device module has a ``wait_for_public_IPv`` parameter. If set to 4 or 6, Ansible will wait until the GET API call for a device will contain an Internet-routeable IP address of the respective version. If we want to refer to an IP address in consequent module calls, it's wise to use the ``wait_for_public_IPv`` parameter.
```
Comment (dharmabumstead) [inline docs/docsite/rst/guide_packet.rst]: Thanks @t0mk! A few review comments:

- Be explicit about what the parameter is. Suggest revising the first sentence to something like "The ``wait_for_public_IPv`` parameter is used to specify the IP version of the IP address to wait for. This value can be 4 or 6."
- s/"will contain"/"contains"
- "the respective version" is a bit hard to parse. Please rephrase.
- Please revise "If we want to..." to something like "To refer to an IP address is subsequent module calls..."

**Review by dharmabumstead — CHANGES_REQUESTED** [review]

File: docs/docsite/rst/guide_packet.rst (hunk 2 above)
Comment (t0mk) [reply docs/docsite/rst/guide_packet.rst]: fixed

**Comment by t0mk** [comment]
Thanks for your reviews. Let me remind you that while we discuss wording and backticks here, the Packet guide on the official docsite is incorrect for more than 2 months now. It's *really* hard to keep support in Ansible consistent.

**Review by gundalow — APPROVED** [review]

**Comment by gundalow** [comment]
Adding to 2.4 project as this should be fixed there as well. I'm happy now.

**Comment by dharmabumstead** [comment]
Updates to the docs need to be clear, concise, and intelligible; hence the care taken over backticks and wording. We *really* appreciate your contributions and your patience - thank you, @t0mk!
